airflow/providers/google/marketing_platform/example_dags/example_campaign_manager.py

```diff
@@ -97,3 +127,41 @@
     # [END howto_campaign_manager_delete_report_operator]
 
     create_report >> run_report >> wait_for_report >> get_report >> delete_report
+
+    # [START howto_campaign_manager_insert_conversions]
+    insert_conversion = GoogleCampaignManagerBatchInsertConversionsOperator(
+        task_id="insert_conversion",
+        profile_id=PROFILE_ID,
+        conversions=[CONVERSION],
+        encryption_source="AD_SERVING",
+        encryption_entity_type="DCM_ADVERTISER",
+        encryption_entity_id=6440317,
```

Comment:
The encryption_entity_id is considered sensitive information and should not be hardcoded.  Can we update this to read from an environment variable?